### DISCUSSION TITLE
Bug when exporting in .gif format

### DISCUSSION THREAD

**Discussion by AtaraxyInDev** [Orama-Interactive/Pixelorama#1098] [Q&A]
I was making a very simple idle animation, and when I exported it in .gif to show it to a friend, some artifacts appeared in the result, I will attach a video to show you if it is my problem or the program's problem.

https://github.com/user-attachments/assets/6789f43e-4af5-457b-8ef5-efbf7ea51027

**Comment by Variable-ind** [comment]
> I was making a very simple idle animation, and when I exported it in .gif to show it to a friend, some artifacts appeared in the result, I will attach a video to show you if it is my problem or the program's problem.
> 2024-09-01.15-48-26.mp4

Hy can you check and verify that this happens in 1.0.2 as well?

**Reply by AtaraxyInDev** [reply]
Sure! I'm going to try

**Reply by AtaraxyInDev** [reply]
@Variable-ind Artifacts continue to appear

https://github.com/user-attachments/assets/f3a7f489-fcc0-4d4a-b984-4618a3fadba2

**Reply by AtaraxyInDev** [reply]
additionally, if you add a background, the artifact does not appear.
![pj_modelo_Animado_Idle](https://github.com/user-attachments/assets/2995d182-56d1-4327-aeac-5303b9fe71be)

**Comment by NIyueeE** [comment]
same, still some problems here even now. 
I use other application to generate gif.
hope be fixed
